Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -151,10 +148,13 @@ public bool Contains(string key)
         public string[] GetAll(string key)
```

Comment:
See [Comment Policy](https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/commenting-policy.md).UPDATES rule - here we have 

```
        /// <param name="key">Name of the argument. If it is not found, an alternative argument name with additional '-' prefix will be checked.</param>
```

And this does not really reflect the reality after this PR update.